data-products/_docs/data-dictionary.md

```diff
@@ -0,0 +1,113 @@
+# The data dictionary
+
+The `02-data-dictionary.yml` file should contain a detailed specification of the structure of the data in YAML format. Like most schemas, we require that you supply table names, column names and column types, along with other metadata. **Unlike** some other schemas, we also ask that you **describe** the tables and columns so that users understand what they are.
```

Comment:
Tables! Tables! Tables! :-D 
Am I being silly trying to avoid making references to table like structure?

Reply:
Not silly, I agree we'd like to encourage domain and product thinking. This was written from the POV of someone familiar with DBs and schemas, but I can tweak this to include domains.